srv/components/system/logrotate/files/etc/logrotate_hw.d/provisioner.conf

```diff
@@ -18,6 +19,7 @@
 }
 
 /var/log/salt/key {
+    su root prvsnrusers
```

Comment:
why we need that here?

Reply:
i have answered over the group chat.
please discuss here if you have ore queries